source/Calamari.AzureServiceFabric/Calamari.AzureServiceFabric.csproj

```diff
@@ -5,12 +5,14 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <IsPackable>false</IsPackable>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net462</TargetFramework>
+    <TargetFrameworks>net462;net6.0-windows</TargetFrameworks>
+    <RuntimeIdentifiers>win-x64</RuntimeIdentifiers>
+    <LangVersion>8.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Identity.Client" Version="4.48.1" />
-    <PackageReference Include="Microsoft.ServiceFabric" Version="5.4.145" />
```

Comment:
It is very important that this version of the NuGet package exactly matches the version of the Runtime & SDK that you have installed locally. If they are different, you will likely receive COM errors.

Also, the runtime and SDK installed on TeamCity build agents also need to match this value.

Reply:
So are we saying that customers will need specifically `10.1.2338` installed locally?
It might be worth adding a comment in here to make sure future us are aware that we dont break things.

Reply:
@zentron I _think_ so. I think it's going to depend on what changes there are between the SDK & the runtime. The frustrating issue with this is that there is some magic COM stuff going on, so it _might_ work, but I guess if the runtime changes don't match, then you might get COM issues